code.gs

```diff
@@ -1659,6 +1659,38 @@ var i18n = {
     'It looks like you are using an outdated version of this script': 'Parece que tens uma versão desatualizada deste script',
     'You can find the latest one here': 'Podes encontrar a última versão aqui',
   },
+  'th': {
+    'Age': 'อายุ',
+    'Years': 'ปี',
+    'Events': 'อีเวนท์',
+    'Birthdays today': 'วันเกิดวันนี้',
+    'Birthdays tomorrow': 'วันเกิดพรุ่งนี้',
+    'Birthdays in {0} days': 'วันเกิดในอีก {0} วัน',
+    'Anniversaries today': 'วันครบรอบวันนี้',
+    'Anniversaries tomorrow': 'วันครบรอบพรุ่งนี้',
+    'Anniversaries in {0} days': 'วันครบรอบในอีก {0} วัน',
+    'Custom events today': 'อีเวนท์ที่กำหนดเองวันนี้',
+    'Custom events tomorrow': 'อีเวนท์ที่กำหนดเองวันพรุ่งนี้',
+    'Custom events in {0} days': 'อีเวนท์ที่กำหนดเองในอีก {0} วัน',
+    'Hey! Don\'t forget these events': 'เฮ้! อย่าลืมอีเวน์เหล่านี้ล่ะ',
+    'version': 'เวอร์ชั่น',
+    'dd-MM-yyyy': 'วว-ดด-ปปปป',
```

Comment:
This particular string should not be "translated" into other alphabets, but instead it should be used to alter the format of the date.

For example in Italy we usually write the date as DAY/MONTH/YEAR, so this string should is "dd/mm/yyyy" in the italian translation. In the USA instead the usually write it as MONTH/DAY/YEAR, so the translation for this string should read "mm/dd/yyyy".

Please reorder the parameters of this string to adapt to whatever standard is used in your country.

Reply:
O sorry, got it, will fix this.